src/Common/src/Interop/Windows/NtDll/Interop.SECURITY_QUALITY_OF_SERVICE.cs

```diff
@@ -0,0 +1,42 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class NtDll
+    {
+		/// <summary>
+        /// <a href="https://msdn.microsoft.com/en-us/library/windows/desktop/aa379574.aspx">SECURITY_QUALITY_OF_SERVICE</a> structure.
+        /// Contains information used to support client impersonation. A client can specify this information when it connects to a server.
+        /// </summary>
+		[StructLayoutAttribute(LayoutKind.Sequential)]
+        internal unsafe struct SECURITY_QUALITY_OF_SERVICE
+        {
+			/// <summary>
+            /// Specifies the size, in bytes, of this structure.
+            /// </summary>
+            internal UInt32 length;
+			
+			/// <summary>
+            /// Specifies the information given to the server about the client, and how the server may represent, or impersonate, the client.
+            /// </summary>
+            [MarshalAs(UnmanagedType.I4)]
```

Comment:
This is redundant and might(?) make this unblittable. If you're going to use the field, define the enum. https://msdn.microsoft.com/en-us/library/windows/desktop/aa379572.aspx.  Even if you don't explicitly use it, consider defining it anyway for clarity.

Reply:
I added this file as it was being used as part of the Framework implementation, since it is not being used as part of the CoreFx implementation, do you suggest removing this file altogether ?